README.md

```diff
@@ -219,47 +219,42 @@ let () = printGithubRepos ()
 ## Node-specific
 
 #### Read lines from a text file
-
+Uses [bs-node](https://github.com/BuckleTypes/bs-node)
 ```ml
 let () =
-    Node_fs.readFileSync "README.md" `utf8
+    Node.Fs.readFileAsUtf8Sync "README.md"
     |> Js.String.split "\n"
     |> Array.iter Js.log
 ```
 
 #### Read and parse a JSON file
-Uses [bs-json](https://github.com/BuckleTypes/bs-json)
+Uses [bs-json](https://github.com/BuckleTypes/bs-json) and [bs-node](https://github.com/BuckleTypes/bs-node)
 ```ml
 let decodeName text =
-  (* parse JSON *)
-  match Js.Json.parseExn text with
-    | obj -> 
-        (* decode the field [name] *)
-        Json.Decode.(field "name" string obj)
-    | exception _ -> failwith ("Error parsing: " ^ text)
+    (* parse JSON *)
```

Comment:
The comments here are probably unnecessary since the code is so clear and self-descriptive already..